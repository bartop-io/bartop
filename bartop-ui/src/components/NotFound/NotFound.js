import React from 'react';
import styled from 'styled-components';

import strings from '../../strings';

const Wrapper = styled.div`
  text-align: center;
  margin-top: 100px;
`;

const NotFound = () => <Wrapper>{strings.pageNotFound}</Wrapper>;
export default NotFound;
