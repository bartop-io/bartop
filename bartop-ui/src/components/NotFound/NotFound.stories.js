import React from 'react';

import { storiesOf } from '@storybook/react';

import NotFound from './NotFound';

storiesOf('NotFound', module).add('default', () => <NotFound />);
