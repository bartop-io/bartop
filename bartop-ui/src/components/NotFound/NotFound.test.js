import React from 'react';
import { shallow } from 'enzyme';

import NotFound from './NotFound';

it('matches the snapshot', () => {
  const wrapper = shallow(<NotFound />);
  expect(wrapper).toMatchSnapshot();
});
