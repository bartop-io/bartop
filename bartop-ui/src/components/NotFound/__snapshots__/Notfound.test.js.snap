// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<styled.div>
  Oops! This page doesn't seem to exist ðŸ˜“
</styled.div>
`;
